math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1009,6 +1011,29 @@ RVec<typename RVec<T>::size_type> Argsort(const RVec<T> &v)
    return i;
 }
 
+/// Return an RVec of indices that sort the input RVec based on a comparison operator
+///
+/// Example code, at the ROOT prompt:
+/// ~~~{.cpp}
+/// using namespace ROOT::VecOps;
+/// RVec<double> v {2., 3., 1.};
+/// auto sortIndices = Argsort(v, [](double x, double y) {return x > y;});
+/// sortIndices
+/// // (ROOT::VecOps::RVec<unsigned long> &) { 1, 0, 2 }
+/// Take(v, sortIndices);
+/// // (ROOT::VecOps::RVec<double> &) { 3., 2., 1. }
```

Comment:
Simplified a bit:

```suggestion
/// auto sortIndices = Argsort(v, [](double x, double y) {return x > y;})
/// (ROOT::VecOps::RVec<unsigned long> &) { 1, 0, 2 }
/// Take(v, sortIndices)
/// (ROOT::VecOps::RVec<double> &) { 3., 2., 1. }
```

Reply:
Good idea! Do we want the printout of the type and data (like in the ROOT prompt) as code or commented with the `//` in front?

Reply:
let's keep the `//` for consistency with the rest